os/trusted-computing-hardware-requirements.md

```diff
@@ -0,0 +1,58 @@
+# Checking hardware and firmware support for CoreOS Trusted Computing
+
+Trusted Computing requires support in both system hardware and firmware. This document specifies the required support and explains how to determine if a physical machine has the features needed to enable Trusted Computing in CoreOS.
+
+## 1. Check for Trusted Platform Module
+
+Trusted Computing depends on the presence of a Trusted Platform Module, commonly referred to as a TPM. The TPM is a motherboard component responsible for storing the state of the system boot process, and providing a secure communication channel over which this state can be verified. To check for the presence of a TPM, install the latest Alpha version of CoreOS and try to list the TPM device file in the `/sys` system control filesystem:
+
+`# ls /sys/class/tpm/tpm0`
+
+If this returns an error, your system either does not have a TPM, or it is not enabled in the system firmware. Firmware configuration varies by system. Consult your vendor documentation for details.
+
+## 2. Check TPM version
+
+Only version 1.2 TPMs are supported at present. Read the TPM device ID file to discover the TPM version:
+
+`# cat /sys/class/tpm/tpm0/device/id`
+
+If this prints `TRUESTRING`, then the system has a version 1.2 TPM and supports Trusted Computing.
```

Comment:
mjg59 notes in slack that while FALSESTRING is reliably `MSFT0101`, indicating an unsupported v2 TPM, TRUESTRING, indicating a supported v1.2 TPM, is not so reliably patterned. Step 2 may need to be a negative test (as he originally wrote it).